python/llm/example/GPU/README.md

```diff
@@ -14,9 +14,13 @@ This folder contains examples of running BigDL-LLM on Intel GPU:
 - Intel Arc™ A-Series Graphics
 - Intel Data Center GPU Flex Series
 - Intel Data Center GPU Max Series
+- Intel iGPU
+- Intel Arc™ A300-Series or Pro A60
+- Intel dGPU Series
 
 **Operating System**:
 - Ubuntu 20.04 or later (Ubuntu 22.04 is preferred)
+- Windows 10/11, with or without WSL (Verified for HF-Transformers-AutoModels and PyTorch-Models on Intel iGPU and dGPU)
```

Comment:
For HF-Transformers-AutoModels and PyTorch-Models, we also supported for Intel iGPU and dGPU on Windows. => We may put this in another section. May separate system support to Linux / Windows